operators/rx.go

```diff
@@ -19,6 +20,7 @@ var _ rules.Operator = (*rx)(nil)
 
 func newRX(options rules.OperatorOptions) (rules.Operator, error) {
 	data := options.Arguments
+	data = fmt.Sprintf("(?sm)%s", data)
```

Comment:
There isn't a programattic way to do it with Go but this seems like the way. Note I verified that multiple instances of `(?sm)` etc work so even if the input expression from CRS were to change to include options in the future, it wouldn't break.

Reply:
Thanks Rag, now it's clear how 942522 was supposed to match against 942522-7! 
I would Maybe just add a comment about what this line tweak is for?

Reply:
While I am not a big fan of mutating user input I believe there is no other way to achieve this as even Go people recommends this https://groups.google.com/g/golang-nuts/c/jiVdamGFU9E